lib/grape/jsonapi/entity/resource.rb

```diff
@@ -28,6 +28,30 @@ def included_exposure
               end
             end
           end
+
```

Comment:
We use the word `exposure` a lot in this file 😵

Reply:
I'm feeling a bit exposed lol